projects/fishtornado_xr/FishTornado.cpp

```diff
@@ -1258,57 +1271,37 @@ void FishTornadoApp::SetupMetrics()
         return;
```

Comment:
Does this PR address TODO above the SetupMetrics function?

Reply:
Again, subsequent PRs address this.